src/databricks/labs/ucx/hive_metastore/table_migrate.py

```diff
@@ -489,3 +491,118 @@ def _sql_alter_from(self, table: Table, target_table_key: str, ws_id: int):
             f"('upgraded_from' = '{source}'"
             f" , '{table.UPGRADED_FROM_WS_PARAM}' = '{ws_id}');"
         )
+
+
+class ACLMigrator:
+    def __init__(
+        self,
+        table_crawler: TablesCrawler,
+        grant_crawler: GrantsCrawler,
+        ws: WorkspaceClient,
+        workspace_info: WorkspaceInfo,
+        backend: SqlBackend,
+        group_manager: GroupManager,
+        migration_status_refresher: MigrationStatusRefresher,
+        principal_grants: PrincipalACL,
```

Comment:
```suggestion
        principal_acl: PrincipalACL,
```
be consistent with names